docs/contributing.md

```diff
@@ -320,3 +320,20 @@ you can submit a PR based on the `gh-pages` branch of
 
  The `.rs` files generated from `.proto` files are included in the repository,
  and there is a GitHub CI check that will complain if they do not match.
+
+## Profiling
+
+One easy-to-use sampling profiler
+is [samply](https://github.com/mstange/samply). For example:
+```shell
+cargo install samply
+samply record jj diff
+```
+Then just open the link it prints.
+
+Another option is to use the instrumentation we've added manually (using
+`tracing::instrument`) in various places. For example:
+```shell
+JJ_TRACE=/tmp/trace.json jj diff
+```
+Then go to `chrome://tracing` in Chrome and load `/tmp/trace.json` from there.
```

Comment:
Nit: `chrome://tracing` says it is deprecated in favor of perfetto.dev. 

> **July 2022 update**: [chrome://tracing](https://chromium.googlesource.com/catapult/+/refs/heads/main/tracing/docs/chrome%3A//tracing) is deprecated, and by default will redirect to https://ui.perfetto.dev/. At the moment, it‘s still possible to use the old UI, but there’s no guarantee that it will continue to function; please file [feature requests](https://perfetto.dev/docs/#bugs) for any blockers that prevent you from migrating from [chrome://tracing](https://chromium.googlesource.com/catapult/+/refs/heads/main/tracing/docs/chrome%3A//tracing) to Perfetto.

Reply:
Fixed, thanks. That's nice to have a link that's not specific to Chrome (I'm guessing).